stix_shifter_modules/vectra/stix_transmission/connector.py

```diff
@@ -223,6 +223,16 @@ def get_results_data(self, response_dict):
 
             detection_type = record.get('detection_type', '')
 
+            # adding new field for confidence field mapping
+            if 'certainty' in record:
+                record['certainty_score'] = record['certainty']
```

Comment:
do you still need this `certainty_score` field?

Reply:
Yes, certainty_score field is required. 
Previously, the certainty field was mapped to both x-ibm-finding:confidence and x-ibm-ttp-tagging:confidence fields.
As per the schema definition x-ibm-ttp-tagging:confidence range is 0-1 and x-ibm-finding:confidence range is 1-100.
From the vectra datasource, we are getting confidence value in the range 0-100.
x-ibm-ttp-tagging:confidence requires a transformer to convert confidence value in the range 0-100 to 0-1.
Since the same field was mapped to two different stix fields,  when transformer was applied to one field it was applied to both the fields when tested from CP4S.
Hence to avoid this, different field certainty_score is created for mapping x-ibm-finding:confidence.

Reply:
i dont think you need this when you revert the mapping change. see https://github.com/opencybersecurityalliance/stix-shifter/pull/1581/files#r1348755907

Reply:
As per your suggestion removed the code